cli.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
the purpose and usage of this file should be defined somewhere. You put information in your PR description, but it would probably be useful to also include a README or something similar with that information.

Reply:
readme updated